### PR TITLE
Updated results lablog and files in clean in sarek templates

### LINKED ISSUE #229: Change scratch-copy default queue from idx
Right now, the default queue to copy files using scratch is `middle_idx`. It would be quite helpful to change it to middle_obx since its less used and saturated. 

https://github.com/BU-ISCIII/buisciii-tools/blob/d95b73843409fd451296bdff4407b21f9d8b3831/bu_isciii/conf/configuration.json#L55-L58

Either way, an extra optional parameter called "--queue" or "--partition" could be introduced to either use idx or obx in case one of the queues is full.

### LINKED ISSUE #234: Create results lablog for exometrio template and update picard-results filename
The results should include the following files: `multiqc_report.html`, `variants_annot_filterAF_head.tab`, `hsMetrics_all.out` (which should be renamed to **`"mapping_metrics.csv"`** in the corresponding `99-stats/lablog `in all exome/trio templates) and a folder named `vep_annot_tables/` which should contain the tables created with VEP (vep_annot*.tab)

Also  `ANALYSIS/DATE_EXOME01/lablog` should include some lines to generate a script to delete junk files, such as these:
```
scratch_dir=$(echo $PWD | sed "s/\/data\/bi\/scratch_tmp/\/scratch/g")
echo "srun --partition short_idx --chdir $scratch_dir rm -rf work &" > _02_clean.sh
echo "srun --partition short_idx --chdir $scratch_dir rm -rf 01-sarek/gatk4/ &" >> _02_clean.sh´´´

### PR DESCRIPTION
**Pull request by Shettland**
## PR checklist

- [X] This PR provides necessary changes to exomeeb, exometrio and wgstrio templates: Included a RESULTS/lablog, and a secondary script in analysis lablog to delete junk files. Also updated the files to clean in `services.json`. closes #234 
Less important, I changed the default queue of scratch copy to middle_obx. closes #229 
- [ ] Make sure your code lints (`black and flake8`).
- If a new tamplate was added make sure:
    - [X] Template's schema is added in `templates/services.json`.
    - [X] Template's pipeline's documentation in `assets/reports/md/template.md` is added.
    - [X] Results Documentation in `assets/reports/results/template.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).
- [ ] If you know a new user was added to the SFTP, make sure you added it to `templates/sftp_user.json`

### CODE REVIEW

File: bu_isciii/templates/exomeeb/ANALYSIS/ANALYSIS01_EXOME/lablog [hunk 1]
```diff
@@ -46,5 +46,6 @@ EOF
 
 echo "sbatch sarek.sbatch" > _01_run_sarek.sh
 
-echo "srun --partition short_idx --chdir $scratch_dir rm -rf work &" > _02_clean.sh
-echo "srun --partition short_idx --chdir $scratch_dir rm -rf 01-sarek/gatk4/ &" >> _02_clean.sh
\ No newline at end of file
+echo "srun --partition short_obx --chdir $scratch_dir rm -rf work &" > _02_clean.sh
+echo "srun --partition short_obx --chdir $scratch_dir rm -rf 01-sarek/gatk4/ &" >> _02_clean.sh
+echo "srun --partition short_obx --chdir $scratch_dir rm -rf 01-sarek/preprocessing/markduplicates &" >> _02_clean.sh
```
Comment (saramonzon) [inline bu_isciii/templates/exomeeb/ANALYSIS/ANALYSIS01_EXOME/lablog:51]: You said this was easily removed using bu-isciii tools clean? Or did i misunderstood?

File: bu_isciii/templates/exomeeb/RESULTS/lablog_exome_results
Comment (saramonzon) [inline bu_isciii/templates/exomeeb/RESULTS/lablog_exome_results:1]: this should be lablog_exomeeb_results

File: bu_isciii/templates/services.json [hunk 2]
```diff
@@ -264,7 +264,7 @@
       "description": "",
       "clean": {
         "folders":["01-sarek/preprocessing/markduplicates", "02-postprocessing"],
```
Comment (saramonzon) [inline bu_isciii/templates/services.json:266]: decide whether markduplicates is deleted using lablog scirpt or bu-isciii tools

**Comment by saramonzon** [comment]
Ah! And changelog is missing!

File: bu_isciii/templates/exomeeb/ANALYSIS/ANALYSIS01_EXOME/lablog (hunk 1 above)
Comment (Shettland) [reply bu_isciii/templates/exomeeb/ANALYSIS/ANALYSIS01_EXOME/lablog:51]: True, i forgot to remove it

**Review by saramonzon — APPROVED** [review]

**Pull request merged**